Source/RealtimeMeshComponent/Public/RenderProxy/RealtimeMeshGPUBuffer.h

```diff
@@ -214,7 +214,7 @@ namespace RealtimeMesh
 #endif
 				}
 				
-				/*Batcher.QueueUpdateRequest(VertexBufferRHI, UpdateData->GetNumElements() > 0? UpdateData->GetBuffer() : nullptr);
```

Comment:
Starting a block comment is not allowed in a block comment

Reply:
Similar change in https://github.com/TriAxis-Games/RealtimeMeshComponent/pull/292/files